### ISSUE TITLE
Differents results with GPU Nvidia P4000 (8GB) and GPU Nvidia P100 (16GB)

### ISSUE THREAD

**Issue by felipetobars** [matterport/Mask_RCNN#2749]
I have trained a model from pre-trained coco weights, with approximately 960 training images and 320 validation images, applying data augmentation with imgaug and different training stages ('heads', '4 +', 'all' ). I did the training both on a pc with a P4000 GPU  and in google colaboratory with a P100 GPU. The results showed overfitting in the Google Colab environment.
The results are not much better with the GPU P4000 but I get better training and validation loss graphs, but my question is why are not better the results with a GPU with greater graphics memory capacity? I have tried changing the parameters of images per GPU, steps per epoch, decreasing the learning rate, but I still get similar results where the value of loss of validation begins to increase in the last training epochs with the P100. Are there any parameters that Mask R-CNN adjusts automatically according to computational capacity?

Google Colab (Tesla P100):
![image](https://user-images.githubusercontent.com/55366335/147952498-2b838871-52aa-4c4e-9fc5-b86ba03ce5a0.png)
![image](https://user-images.githubusercontent.com/55366335/147952522-c58ea61b-6e9c-456b-880d-0db210b6bfe4.png)
Nvidia P4000:
![image](https://user-images.githubusercontent.com/55366335/147952556-4f8e0764-57db-486a-94de-1ef6a6d6ce15.png)
![image](https://user-images.githubusercontent.com/55366335/147952564-95b9d8ea-f895-40d9-9ea3-66d5c62b515f.png)

I appreciate your responses.

**Comment by monjurulkarim** [comment]
@felipetobars I also faced with similar problem that I couldn't solve yet. Hope, someone in this forum may help us out.
